components/sdlutil/sdlcursormanager.cpp

```diff
@@ -22,6 +25,13 @@
 USE_GRAPHICSWINDOW()
 #endif
 
+#if OSG_VERSION_GREATER_OR_EQUAL(3, 5, 8) || defined(__APPLE__)
```

Comment:
It seems that only Apple has this issue right, so why are we applying this if OSG >= 3.5.8 OR APPLE?

Shouldn't this be && instead of || ?